### ISSUE TITLE
Cannot resolve 1.1.0 dependencies

### ISSUE THREAD

**Issue by Chaseos** [ouattararomuald/android-image-slider#36]
I've tried resolving any of the 1.1.0 dependencies in Gradle and only get "Failed to resolve" errors. The only one I found that works is the first version (com.ouattararomuald:slider:1.0.0).

**Comment by ouattararomuald** [comment]
Oh sorry for that !
The release is `1.0.1` (AndroidX) compatible;

`1.1.0` is only available in snapshot (not released yet)

**Comment by Chaseos** [comment]
`1.0.1` dependencies don't seem to be working either. Still only `1.0.0` through Gradle.

**Comment by ouattararomuald** [comment]
I just grabbed it through Gradle and everything works fine : 

```gradle
implementation 'com.ouattararomuald:slider:1.0.1'
```

Please make sure you've `mavenCentral()` in your repositories and if you can make sure it's above jcenter and jitpack.

```gradle
repositories {
  google()
  mavenCentral()
  jcenter()
}
```

**Comment by Chaseos** [comment]
I started brand new projects in Android Studio Stable and Canary and added the `mavenCentral()` into both locations of repositories, tried all three of the implementations 
```        
    implementation 'com.ouattararomuald:slider-glide:1.0.1'
    implementation 'com.ouattararomuald:slider-picasso:1.0.1'
    implementation 'com.ouattararomuald:slider:1.0.1'
```
and they all ended up with a `ERROR: Failed to resolve`. Switching `implementation 'com.ouattararomuald:slider:1.0.1'` back to `implementation 'com.ouattararomuald:slider:1.0.0`allows me to build with it again.

**Comment by ouattararomuald** [comment]
This is my fault ! I accidentally released the README of the next version `1.1.0`.
`1.0.1` had no glide or picasso dependency.
